src/mongo/db/pipeline/TDigest.h

```diff
@@ -19,10 +19,10 @@
 #include <cmath>
 #include <vector>
 
-#include <folly/Range.h>
-#include <folly/Utility.h>
+struct presorted_t {};
```

Comment:
Now that we changed the 'merge' signature a little to only accept presorted values those two definitions can go away.